.circleci/config.yml

```diff
@@ -35,11 +35,40 @@ jobs:
       - run:
           name: tests
           command: yarn test
+      - persist_to_workspace:
+          root: ~/project
+          paths:
+            - .
+
+  build-ubuntu:
+    docker:
+      - image: electronuserland/builder:14
+    working_directory: ~/project
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: install dpkg and fakeroot
+          command: |
+            apt-get update -y
+            apt-get install -y dpkg fakeroot
+      - run:
+          name: generate builds
+          command: yarn make
+      - store_artifacts:
+          path: ~/project/out/make
 
 workflows:
-  lint-and-test:
+  lint-test-package:
     jobs:
       - install-deps
       - lint-and-test:
           requires:
             - install-deps
+      - build-ubuntu:
+          filters: &filters-build
+            branches:
+              only:
+                - master
```

Comment:
We should be building only for `release/*` branches. Also, please consider adding an automatic artifact upload to a draft release (perhaps in another PR, seems to be complex enough)

Reply:
ok

Reply:
https://github.com/pastelnetwork/pastel-electron-wallet/pull/107/commits/26f72d4701afd75eac773991079a2159047808ad